mmv1/third_party/tgc/tests/data/example_google_dataflow_job.json

```diff
@@ -0,0 +1,19 @@
+[
+    {
+        "name": "//dataflow.googleapis.com/v1b3/projects/{{.Provider.project}}/locations/placeholder-2EsS2bm9/jobs",
+        "asset_type": "dataflow.googleapis.com/Job",
+        "ancestry_path": "{{.Ancestry}}/project/{{.Provider.project}}",
+        "resource": {
+            "version": "v1b3",
```

Comment:
Can we make this same as CAI list assets response which is "v1beta3"

Reply:
done, thanks